uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
```

Comment:
> I'm pretty sure I hit this myself during testing with some bewildering cached data that magically went away when I restarted the server (didn't realized it was a cached_property til later).

To be clear, the cached property is set on ``self`` and not on ``self.__class`` so it's cached per-instance and not across all instances.  Restarting the server shouldn't affect a cached property because our model instances go stale as soon as their session closes.